test/topology_custom/test_major_compaction.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import pytest
+import logging
+import asyncio
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.log_browsing import ScyllaLogFile
+
+logger = logging.getLogger(__name__)
+
+async def disable_autocompaction_across_keyspaces(manager: ManagerClient, server_ip_addr: str, *keyspace_list):
```

Comment:
Don't we already have such a function?
There is definitely one in cql-pytest...

Any reason this test is in topology_custom?

Reply:
The test checks the log to see if the tables are being flushed using calls like `log.mark()`. I tried to do this inside `cql-pytest` but I'm unable to access log from inside that suite. `topology-custom` is able to do this as it controls the scylla server via the ManagerClient object. The test also starts the server with different `compaction_flush_all_tables_before_major_seconds` values and checks if the tables get flushed - this is easier to do from the topology suite.

Reply:
There's also the system.compaction_history table.

Reply:
I needed to check if all the tables got flushed. `system.compaction_history` did not have that information. This patch adds a new log line to the `flush_all_tables` method and greps that from the testcase to check if the tables were flushed.